<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowResizeInterface</class>
 <widget class="QDialog" name="WindowResizeInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planos de Sala BETA y REUNIONES</string>
  </property>
  <widget class="QPushButton" name="CLOSE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameR">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
